frontend/next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
I didn't make this change. Wondering if this happened when I ran the linter/formatter.